src/sections/Pricing/Pricing.tsx

```diff
@@ -26,64 +27,91 @@ const DescriptionComponent = styled.div`
   margin-bottom: 32px;
 `;
 
+type PricingCard = {
+  color: string;
+  title: string;
+  price?: string;
+  denominator?: string;
+  benefits: string[];
+  buttonText: string;
+  buttonLink: string;
+  isMostPopular?: boolean;
+  btnClassName: string;
+};
+
+const pricingCards: PricingCard[] = [
```

Comment:
Do we need this `PricingCard[]` type here?

Reply:
Hm, from my perspective it is more good than harm to have it here. It makes it more clear what we are expecting in this array, it also prevents misspelling or adding something that is not expected by `PricingCard` component. To be honest I would leave it but also I think can be quite overzealous when it comes to types so if you are strongly convinced that it is redundant say without any hesitation! 🙈